vendor-bin/churn-php/composer.json

```diff
@@ -1,5 +1,10 @@
 {
     "require-dev": {
         "bmitch/churn-php": "^1.3"
+    },
+    "config": {
+        "allow-plugins": {
+            "composer/package-versions-deprecated": true
```

Comment:
when run composer test, these liens are added ?

Reply:
You can let it as it is.